### PR TITLE
BFD-1506: add details to tls cert pages

### PR DESCRIPTION
**Pull request by keithdadkins**
**JIRA Ticket:**
[BFD-1506](https://jira.cms.gov/browse/BFD-1506)

**User Story or Bug Summary:**

As a BFD engineer, I want to configure alerts for certificate renewal in Jenkins, so I am alerted when certificates need to be renewed.

---

### What Does This PR Do?

Adds a few key details when getting paged for expiring certificates. Without these details, all the on-call engineer sees is a page containing a random id. 

The "message_state" field, which is supposed to allow long strings, does not appear (at least for me) in either the iphone app or website. However, it's totally possible I'm borking the REST, will test tomorrow.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Review by mackec1 — DISMISSED** [review]

File: ops/ccs-ops-misc/cert-checker/cert-checker.sh [hunk 1]
```diff
@@ -9,9 +9,9 @@ SEND_NOTIFICATIONS=${SEND_NOTIFICATIONS:-"false"}
 CRON_SCHEDULE="${SCHEDULE:-"0 12 * * *"}" # run at noon everyday
 
 # notifications
-WARN_WHEN="${WARN_WHEN:-"3000:4500"}"     # WARN when expiration is >= 30 but less than 45 days
-ALERT_WHEN="${ALERT_WHEN:-"1500:3000"}"   # expiration is >= 15 but less than 30 days
-PAGE_WHEN="${PAGE_WHEN:-"1:1500"}"        # expiration is >= 0 but less than 15 days
+WARN_WHEN="${WARN_WHEN:-"30:45"}"     # WARN when expiration is >= 30 but less than 45 days
+ALERT_WHEN="${ALERT_WHEN:-"15:30"}"   # expiration is >= 15 but less than 30 days
+PAGE_WHEN="${PAGE_WHEN:-"1:15"}"      # expiration is >= 0 but less than 15 days
```
Comment (christrotteradhoc) [inline ops/ccs-ops-misc/cert-checker/cert-checker.sh:14]: should this be 0:15?

Comment (keithdadkins) [reply ops/ccs-ops-misc/cert-checker/cert-checker.sh:14]: Refactored page logic to remove min date.

File: ops/ccs-ops-misc/cert-checker/env.sample [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 # notifications
-export SEND_NOTIFICATIONS="false"     # Send slacks/pages? "true" or "false"
-export WARN_WHEN="3000:4500"              # WARN when expiration is >= 30 but less than 45 days
-export ALERT_WHEN="1500:3000"             # ALERT when expiration is >= 15 but less than 30 days
-export PAGE_WHEN="1:1500"               # PAGE when expiration >= 0 but less than 15
+export SEND_NOTIFICATIONS="false"   # Send slacks/pages? "true" or "false"
+export WARN_WHEN="3000:4500"        # WARN  when >= 30 and < 45 days
+export ALERT_WHEN="1500:3000"       # ALERT when >= 15 and < 30 days
+export PAGE_WHEN="1500"             # PAGE  when <= 15 days
```
Comment (christrotteradhoc) [inline ops/ccs-ops-misc/cert-checker/env.sample:10]: these seem like they should be 30, 45, etc and not 3000, 4500, etc. ?

Comment (keithdadkins) [reply ops/ccs-ops-misc/cert-checker/env.sample]: Fixed

**Review by christrotteradhoc — APPROVED** [review]

**Review by cbrunefearless — APPROVED** [review]
LGTM

**Pull request merged**